# same effect as patch for futile (futile_index_bug.patch), except due to directory structure, must be regenerated
# fix the redefinition of an index variable caught by GCC 11
# Author: Denis Kristak (Inuits)
diff -ruN bigdft-suite-1.9.1_orig/futile/flib/f_input_file.f90 bigdft-suite-1.9.1/futile/flib/f_input_file.f90
--- bigdft-suite-1.9.1_orig/futile/flib/f_input_file.f90	2022-02-22 10:55:06.924505000 +0100
+++ bigdft-suite-1.9.1/futile/flib/f_input_file.f90	2022-02-22 10:55:57.462690000 +0100
@@ -172,7 +172,6 @@
     type(dictionary), pointer :: dict
     character(len = *), intent(in) :: file, key
 
-    integer :: i
     type(dictionary), pointer :: ref,iter
     character(len = max_field_length) :: val, profile_
     double precision, dimension(2) :: rg
@@ -366,6 +365,7 @@
       character(max_field_length), dimension(:), allocatable :: keys
       double precision :: var
       integer :: dlen, skeys
+      integer :: i
 
       if (associated(dict%child)) then
          if (dict_len(dict) >= 1) then
